src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNodeTests.cs

```diff
@@ -342,6 +343,132 @@ public void TestGetAllDirectivesUsingDescendantNodes()
             }
         }
 
+        [Fact]
+        public void TestContainsDirective()
+        {
+            // Empty compilation unit shouldn't have any directives in it.
+            for (var kind = SyntaxKind.TildeToken; kind < SyntaxKind.ScopedKeyword; kind++)
```

Comment:
How were these kinds chosen? Is there any risk of these rotting as more syntax kinds are added?

Reply:
It's the start/end kinds.  There's a chance of it rotting... but i don't care. t he main idea here is just to show that teh right directive kinds show up, and other kinds (especially trivia kinds) don't.